frontend/src/Onboarding/GenericOnboarding.tsx

```diff
@@ -90,3 +176,39 @@ export const GenericOnboardingTemplate: React.FC<Props> = ({
     </Wrapper>
   );
 };
+
+const OnboardingSelectionTemplate: React.FC<
```

Comment:
Abstracting out most of the common elements within the transferable credit and completed courses screens